src/Yoti/Util/Profile/AttributeConverter.php

```diff
@@ -0,0 +1,27 @@
+<?php
+namespace Yoti\Util\Profile;
+
+
+use Yoti\Entity\Attribute;
+use Yoti\Entity\DocumentDetails;
+
+class AttributeConverter
+{
+    public static function convertValueBasedOnAttributeName($value, $attrName)
```

Comment:
Why not just pass in the whole attribute?

Reply:
We don't need all the properties in the attribute Object, we only need those 2 params.